install.sh

```diff
@@ -0,0 +1,18 @@
+echo '\ngetting Trust WalletLink...'
+if [ -d src ]; then
+    cd  src
+
+    git pull git@github.com:dpereskokov/walletlink.git
```

Comment:
@vikmeup @hewigovens may be it will be better to have this fork inside Trust github?

Reply:
yes, we can move it to Trust organization, why not submodule? custom shell script doesn't look like a normal js/npm project